plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -160,12 +160,19 @@ private static JSONObject generateRootConfiguratorObject() {
         return rootConfiguratorObject;
     }
 
-    private static void generateMultipleAttributeSchema(JSONObject attributeSchema, Attribute attribute) {
+    private static void generateMultipleAttributeSchema(JSONObject attributeSchema, Attribute attribute, BaseConfigurator baseConfigurator) {
         if (attribute.type.getName().equals("java.lang.String")) {
+            ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
             attributeSchema.put(attribute.getName(),
                 new JSONObject()
                     .put("type", "string"));
         } else {
+            ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
+            try {
+                System.out.println("DocString: " + configurationAsCode.getHtmlHelp(baseConfigurator.getTarget(), attribute.getName()));
```

Comment:
@timja as per thishttps://github.com/jenkinsci/configuration-as-code-plugin/issues/1059#issuecomment-532317607 
The docString seems to return null every single time. I tried changing the target but to no avail.

Reply:
you need the descriptor not the configurator for it to work